CHANGELOG.md

```diff
@@ -14,10 +14,10 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Upgrade NVIDIA driver to version 470.82.01.
 - Upgrade CUDA library to version 11.4.3.
 - Upgrade NVIDIA Fabric manager to 470.82.01.
+- Upgrade Intel MPI Library to 2021.4.0.441.
```

Comment:
Please, consider the comments added to the equivalent PR: https://github.com/aws/aws-parallelcluster-cookbook/pull/1313#pullrequestreview-851970629